### ISSUE TITLE
dcnm_interface  Merged issue

### ISSUE THREAD

**Issue by nabsdcnm** [CiscoDevNet/ansible-dcnm#137]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Ansible Version and collection version

ansible version 2.10
cisco.dcnm 1.2.0  and cisco.dcnm 2.0.1

### DCNM version

* V 11.5.1

### Affected module(s)

* cisco.dcnm.dcnm_interface

### Ansible Playbook

```yaml
#  - name: Add cmd inside freeform config of the interface 
       cisco.dcnm.dcnm_interface: 
        fabric: PR_DCI_FCT
        state: merged
        config:
          - name: "Ethernet1/12"
            type: eth
            switch:
              - "1.2.3.4"
             deploy: false
             profile:
               mode: trunk
               cmds:
                 - 'vpc orphan-port suspend'
     
```

### Expected Behavior

I expected to add a cli command inside interface "freeform config field" even if there is already other cli commands inside this field.

### Actual Behavior

If there is already cli commands inside interface "Freeform Config filed" , the merged state in the playbook replace all the "Freeform Config filed" by the cli in the playbook.

**Comment by mikewiebe** [comment]
@nabsdcnm Thanks for raising the issue.  We are discussing the behavior here and will respond soon.

**Comment by nabsdcnm** [comment]
@mikewiebe, thank for your reply.
Can you please tell me when this fix will be available ? 
Another question, do you know if the merge in the network module has the same issue ?

**Comment by mikewiebe** [comment]
@nabsdcnm We currently do not support a free form set of commands for the `dcnm_network` module.

We don't have an estimated date for this fix yet but we are finishing up the fix for a different issue and then we will take this up next.

**Comment by nabsdcnm** [comment]
Hi,

This issue was closed but we don't know on which version this issue is fixed.
Can we have a reply please ?

Thx a lot for your lights.

Regards.

**Comment by praveenramoorthy** [comment]
@nabsdcnm the fix for  issue is available in 2.1.0 and later releases
